src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/Card/_content.html.twig

```diff
@@ -0,0 +1,15 @@
+<div class="card-content">
+    {% if withPreview is defined %}
+        <i class="grey-text text-darken-4 activator material-icons right">more_vert</i>
+    {% endif %}
+    <a href="{{ path('view', { 'id': entry.id }) }}" title="{{ entry.title| striptags | e('html_attr') }}" class="card-title dot-ellipsis dot-resize-update">
+        {{ entry.title | striptags | truncate(80, true, '…') | raw | default('config.form_rules.faq.variable_description.title'|trans) }}
```

Comment:
Do we need to create a new translation key or use this one?

Reply:
I think we should create a new translation key :+1:

Reply:
Ok, I'll do that with the same translations.

Reply:
Because there is a lot of files to modify, I would like having a validation before editing everything. I thought about adding this to the `entry` section, like this:
```yml
entry:
    default_title: "xxxxxxxx"
```
Is it ok for you?

Reply:
It's ok for me